README.md

```diff
@@ -37,10 +55,31 @@ export ETCDBACKUP_PASSPHRASE=ZZZ
 etcd-backup -aws-s3-bucket bucket -prefix cluster1
 ```
 
+### Create V2 and V3 backup
+
 To create both V2 and V3 make sure etcd data directory accessible locally.
 
 ```
 etcd-backup -aws-s3-bucket $BUCKET_NAME -prefix $CLUSTER_NAME -etcd-v2-datadir /var/lib/etcd
 ```
 
-# TODO
+### Restore backup
+
+To restore backup use following [guide](Documentation/01-restore-etcd-from-backups.md) as example.
+
+## Future Development
+- Implement additional storage backends.
+
+## Contact
+
+- Mailing list: [giantswarm](https://groups.google.com/forum/!forum/giantswarm)
+- IRC: #[giantswarm](irc://irc.freenode.org:6667/#giantswarm) on freenode.org
+- Bugs: [issues](https://github.com/giantswarm/etcd-backup/issues)
+
+## Contributing & Reporting Bugs
+
+See [.github/CONTRIBUTING.md](/giantswarm/example-opensource-repo/blob/master/.github/CONTRIBUTING.md) for details on submitting patches, the contribution workflow as well as reporting bugs.
```

Comment:
Link to Contributing page is broken. I suggest an absolute path required (or adding CONTRIBUTING.md into current repository)

Reply:
Done.